src/elements/BaseButton.vue

```diff
@@ -153,18 +176,24 @@ button:hover > .tooltip,
   }
 
   &.outline {
-    background: var(--colour-neutral-base);
-    border: 0.0625rem solid var(--colour-primary-default);
+    position: relative;
+    background: transparent;
+    --button-outline-border: var(--colour-primary-default);
```

Comment:
Do we want to position CSS custom properties at the top of the rule definition (moving it two lines up in this case)? Imho that might be a bit cleaner.